motion/p5js/conobjects/index.js

```diff
@@ -0,0 +1,78 @@
+// points on the canvas
+var places = [-80, 200, 350 ];
+// number of movers
+var movers = [1,1,1]
+var csize = 500;
+
+function setup(){
+  createCanvas(csize,csize);
```

Comment:
Please assign the canvas to the sketch element https://github.com/fabianmoronzirfas/gestalten-in-code/wiki/Contributing-P5js-Sketches#code